docs/isa-subscriptions/Basics.md

```diff
@@ -23,6 +23,34 @@ Be aware that not all changes to a portfolio mean a change to the corresponding
 
 - Withdrawals
 
+## Flexible ISAs
+
+As of the beginning of the 2025-2026 tax year all ISAs under our management will be flexible, excluding JISAs which are not eligible for flexibility. This means that any existing and all new ISAs added can withdraw from their portfolio and the amount withdrawn will be subtracted from their current year subscription. Later, they can then replace the amount withdrawn to re-subscribe to their ISA. Keep in mind that only transactions of type `Withdrawal` and `InternalCashTransferOut` will allow replacement, and the replaceable amount will be lost if the ISA is transferred to a new provider. Note that, when transferring providers, the new provider may also not offer flexibility.
+
+The 'replaceable amount' will be available over our API where a client has not subscribed, but has withdrawn from their ISA. You can calculate the total allowed amount that a client can contribute by doing the following: `allowance - subscription + replaceableAmount`. For example, if someone has a replaceable amount of £50,000, we will allow them to contribute £70,000 (20,000 - 0 + 50,000) and it will successfully match to their portfolio.
+
+Below are some examples of some different scenarios with a flexible ISA.
+
+### Example 1: An existing ISA has £100,000 and no current year subscriptions
```

Comment:
Is it worth mentioning the first subscribed date on these examples?

Reply:
I don't think that really matters to tenants, as it's more for reporting/transfer documentation (between providers/HRMC)